### DISCUSSION TITLE
CRUD on joined tables [May 04]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2017] [Discord Threads]
> 🙋‍♂️ **Paul**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Yes, that's correct. Does ms SQL support materialised views? A user requested it for  Pg  
#  
> 🙋‍♂️ **Paul**  
#### Yes, materialized views exist in MS SQL, I think they are called indexed views.   
#### Can we see them in NocoDB ?  
#  
> 💁 **o1lab**  
#### sorry, we do not support them. There was a discussion abt the same for PG in github issues  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/970627453196333086)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
